### PR TITLE
CSS changes for Curated cards to remove nesting

### PR DESCRIPTION
**Pull request by skumawat1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #[WRLDBAEMS-131](https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-131)
[WRLDBAEMS-135](https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-135)
[WRLDBAEMS-136](https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-136)

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://cc-issues--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: blocks/curated-cards/curated-cards.css [hunk 1]
```diff
@@ -1,194 +1,173 @@
...
+  .curated-cards.list ul {
+    grid-template-columns: 1fr;
+  }
+
+  .curated-cards.list ul li {
+    grid-template-columns: minmax(15%, 144px) 1fr;
   }
 }
 
 @media (width <=768px) {
```
Comment (mkbansal1) [inline blocks/curated-cards/curated-cards.css:151]: mobile is < 768

File: blocks/curated-cards/curated-cards.css [hunk 2]
```diff
@@ -1,194 +1,173 @@
...
+.curated-cards.list ul li .curated-cards-card-text-wrapper .curated-cards-card-event {
+  font-weight: 700;
+  font-size: var(--body-tiny);
+  letter-spacing: 0.06em;
+  text-transform: uppercase;
+  color: var(--neutrals-50);
+  margin-top: 15px;
 }
 
 @media (width <=1024px) {
```
Comment (mkbansal1) [inline blocks/curated-cards/curated-cards.css:123]: Tab is <1024

**Review by mkbansal1 — CHANGES_REQUESTED** [review]

File: blocks/curated-cards/curated-cards.js [hunk 3]
```diff
@@ -31,21 +31,25 @@ function createFeatureCard(row) {
...
 function processRow(row) {
-  const [imageContent, tagContent, headingContent] = row.children;
+  const [imageContent, tagContent, headingContent, linkDiv] = row.children;
   const liTag = li();
   moveInstrumentation(row, liTag);
   const textWrapper = div({ class: 'curated-cards-card-text-wrapper' });
   const imageDiv = div({ class: 'curated-cards-card-img' });
   const tagElement = div({ class: 'curated-cards-card-event' });
   const heading = p();
+  const link = linkDiv.textContent ? linkDiv.textContent : '';
```
Comment (thomsebastin) [inline blocks/curated-cards/curated-cards.js:51]: I think you can also do `const link = linkDiv.textContent || ''`

**Review by thomsebastin — APPROVED** [review]

File: blocks/curated-cards/curated-cards.js (hunk 3 above)
Comment (skumawat1) [reply blocks/curated-cards/curated-cards.js:51]: will update

File: blocks/curated-cards/curated-cards.css (hunk 2 above)
Comment (thomsebastin) [reply blocks/curated-cards/curated-cards.css]: Using < 1024px would exclude devices with exactly 1024px width, so it's generally safer to use <= 1024px (expressed as max-width: 1024px in CSS) to cover all cases. @mkbansal1

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**